CODE_OF_CONDUCT.md

```diff
@@ -1,139 +1,149 @@
+  4. **Sexualization.** Behaving in a way that would generally be considered inappropriately intimate in the context or purpose of the community.
+  5. **Violating confidentiality.** Sharing or acting on someone's personal or private information without their permission.
+  6. **Endangerment.** Causing, encouraging, or threatening violence or other harm toward any person or group.
+  7. Behaving in other ways that **threaten the well-being** of our community.
+
+### Other restrictions <a name="other-restrictions"></a>
+
+  1. **Misleading identity.** Impersonating someone else for any reason, or pretending to be someone else to evade enforcement actions.
+  2. **Misrepresenting project affiliation.** Speaking or acting in a way that implies an official affiliation with the BeeWare project, where one does not exist.
+  3. **Failing to credit sources.** Not properly crediting the sources of content you contribute.
+  4. **Promotional materials.** Sharing marketing or other commercial content in a way that is outside the norms of the community.
+  5. **Excessive communication.** Disrespecting the time and space of others by engaging in an unacceptable volume of communication.
+  6. **Unhelpful communication.** Offering opinions without relevant experience in the topic being discussed, entering into an ongoing discussion without first gaining familiarity with the history of the topic, or making contributions that are off-topic or otherwise distracting.
+  7. **Irresponsible messaging.** Failing to responsibly present content which includes, links or describes any other restricted behaviors.
+  8. Other conduct that could reasonably be considered **unprofessional or inappropriate**.
+
+## Reporting an issue <a name="reporting-an-issue"></a>
+
+Tensions can occur between community members even when they are trying their best to collaborate. Not every conflict represents a code of conduct violation, and this Code of Conduct reinforces encouraged behaviors and norms that can help avoid conflicts and minimize harm. Reporting even minor issues is important, as they can be helpful in identifying patterns of behavior that may not be concerning in isolation, but when viewed collectively may be more significant.
+
+When an incident does occur, it is important to report it promptly to the BeeWare Code of Conduct Response Team.
+
+**If you believe you or anyone else is in physical danger, please notify appropriate law enforcement first.**
+
+To report a possible violation, email the Team at [conduct@beeware.org](mailto:conduct@beeware.org). If necessary, you can reach out to individual team members. On the BeeWare Discord server, you can also direct message anyone on the Response Team, or, if appropriate, mention @moderators in a public channel. Team members can be reached by the following usernames on Discord or GitHub, or the provided email addresses:
```

Comment:
GitHub's syntax highlighting implies that `@moderators` might be parsed as a user mention, so escape it just in case...

```suggestion
To report a possible violation, email the Team at [conduct@beeware.org](mailto:conduct@beeware.org). If necessary, you can reach out to individual team members. On the BeeWare Discord server, you can also direct message anyone on the Response Team, or, if appropriate, mention \@moderators in a public channel. Team members can be reached by the following usernames on Discord or GitHub, or the provided email addresses:
```